backend/inspirehep/curation/cli.py

```diff
@@ -46,12 +28,17 @@ def update_pdg_keywords(ctx, url):
         record["inspireId"]: record["pdgIdList"] for record in pdg_json_data
     }
     records_with_pdg_keywords_query = Q("match", keywords__schema="PDG")
-    search_obj = (
-        LiteratureSearch()
-        .query(records_with_pdg_keywords_query)
-        .params(size=1000, scroll="60m")
-    )
-    records_with_pdg_recids = (rec["control_number"] for rec in search_obj.scan())
-
-    for batch in chunker(records_with_pdg_recids, 50):
+    scan_obj = (
+        LiteratureSearch().query(records_with_pdg_keywords_query).params(size=1000)
+    ).scan()
+
+    generator_empty = False
+    while not generator_empty:
+        batch = []
+        for _ in range(50):
```

Comment:
could you please remind me why 50?

Reply:
it seemed reasonable (and we used this batch size in other task) but I can make it adjustable with a cli option

Reply:
👍 let's have it as an option